ui/app/routes/vault/cluster/access/mfa/methods/method.js

```diff
@@ -1,8 +1,22 @@
 import Route from '@ember/routing/route';
-
+import { hash } from 'rsvp';
 export default class MfaMethodRoute extends Route {
   model({ id }) {
-    return this.store.findRecord('mfa-method', id);
+    return hash({
+      method: this.store.findRecord('mfa-method', id).then((data) => data),
```

Comment:
what if no record can be found? could that situation happen?

Reply:
After looking into the documentation for `hash` further I think it is fine to use it in a route model hook if the resolved data from **all** of the promises is required for the route. `hash` returns a promise that will be rejected if any of the supplied promises are rejected which will then trigger the error event in the route.

With that said, I think a method can exist on it's own which makes the result of the promise to fetch enforcements not required to load the route. Since you are swallowing the error it shouldn't be a problem here but just wanted to provide a bit of background on this pattern.